pegasus/helpers/prize_helpers.rb

```diff
@@ -1,7 +1,8 @@
 require src_dir 'abort_form_error'
 
 def request_ip
-  defined? request ? request.ip : nil
+  return nil unless defined? request
+  request.ip
 end
 
 def claim_prize_code(type, email, purpose, params={})
```

Comment:
It's not currently included, and since this is called from process_forms the request.ip is always 127.0.0.1. Perhaps we should pass in the ip address from the form.